rules/backend.yar

```diff
@@ -243,6 +243,34 @@ rule md5_f797dd5d8e13fe5c8898dbe3beb3cc5b {
     strings: $ = "echo(\"FILE_Bad\");"
     condition: any of them
 }
+rule md5_22927e670b0953c01874d626110aa474 {
+    strings:
+        $ =  /(RegExp[^\)]*(onepage|nexwaycheckout|checkout|onestep|firecheckout|onestepcheckout)[^\)].+(grelos_v=null))/
```

Comment:
What do you think of this one?

```
((onepage|checkout|firecheckout|nexwaycheckout|onestep|onestepcheckout)\|){2,}
```
(at least 2 of the above, joined with a `|`)